service/src/main/resources/application.yml

```diff
@@ -76,6 +76,11 @@ workspace:
     polling-interval: "15m"
     startup-wait: "5s"
 
+  dangling-resource-cleanup:
+    enabled: true
+    polling-interval: "15m"
```

Comment:
15 min feels like a lot. is there security/cost concern with those dangling resources? can we do daily?

Reply:
curious, where is the output of the job stored?

Reply:
Currently just logging resources that are cleaned up.

Reply:
sorry i mean for a job that's running in a cluster, where does the researchers see the result aftere the cluster is deleted?

Reply:
Oh sorry I misinterpreted, all in the cloud console. Jobs are independent of clusters so you can view all jobs + outputs regardless of whether the clusters that ran them still exist or not,